DiscordLink/Source/WebApi/V1/DiscordLinkController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Eco.WebServer.Web.Authorization;
+using Eco.Shared;
+using Eco.Shared.Utils;
+using DiscordLink.Source.WebApi.V1.Models;
+
+namespace Eco.Plugins.DiscordLink
+{
+    /// <summary>MVC <seealso cref="Controller"/> instance for accessing and interacting with DiscordLink.</summary>
+    [Route("discordlink/api/v1")]
+    [Authorize(Policy = PolicyNames.RequireAdmin)]
+    public class DiscordLinkController : Controller
+    {
+
+        /// <summary>Force restart DiscordLink</summary>
+        [HttpPost("restart")]
+        [Produces("application/json")]
+        public async Task<bool> PostReloadPlugin()
+        {
+            return await SharedCommands.RestartPlugin(SharedCommands.CommandInterface.WebCommand, null);
+        }
+
+        /// <summary>Post an Invite to the discord server.</summary>
+        [HttpPost("chat/invite")]
+        [Produces("application/json")]
+        public async Task<bool> PostInviteMessage()
+        {
+            bool sent = await SharedCommands.PostInviteMessage(SharedCommands.CommandInterface.WebCommand, null);
+            Response.StatusCode = sent ? 201 : 500;
+            return sent;
+        }
+
+        /// <summary>Get the status of Discordlink.</summary>
+        [HttpGet("status/plugin")]
+        [Produces("application/json")]
+        public DiscordLinkPluginStatusV1 GetPluginStatus()
+        {
+            DiscordLink plugin = DiscordLink.Obj;
+            DLDiscordClient client = plugin.Client;
+            return new DiscordLinkPluginStatusV1
+            {
+                DiscordStatus = client.Status,
+                PluginStatus = plugin.GetStatus(),
+                ServerVersion = EcoVersion.VersionNumber,
+                PluginVersion = plugin.PluginVersion.ToString(),
+                ClientName = client.DiscordClient.CurrentUser.Username
+            };
+        }
+
+        /// <summary>Get the status of the current Configuration.</summary>
+        [HttpGet("status/config")]
```

Comment:
We already have the vanilla `api/v1/plugins/config/DiscordLink` endpoint, so i feel like this duplicates some stuff.
This feels like it's just a status, not a config, so `[HttpGet("status")]` feels more consistent